content/backend/graphql-scala/8-mutations.md

```diff
@@ -0,0 +1,315 @@
+---
```

Comment:
Hi Mariusz,
Since I learn about mutations I read this draft. First question I have: https://www.quora.com/Is-it-an-user-or-a-user